cmd/kubeadm/app/phases/upgrade/compute.go

```diff
@@ -279,7 +279,7 @@ func GetAvailableUpgrades(versionGetterImpl VersionGetter, experimentalUpgradesA
 }
 
 func getBranchFromVersion(version string) string {
-	return strings.TrimPrefix(version, "v")[:3]
```

Comment:
I see this works, but I'd like to make it more resilient by using `versionutil` package for parsing and then invoking `v.Minor()`. Can you follow-up a PR with that please?

Reply:
Yeah. Addressed this in #64008.